util/process_perf_data.py

```diff
@@ -239,7 +245,7 @@ def create_treeherder_job(repo, revision, client, nodes, s3=None):
 
         # To avoid overwriting existing data (perhaps if a job is retriggered)
         # the job guid is included in the key.
-        log_prefix = "%s/%s/%s" % (repo, revision, job_guid)
+        log_prefix = "%s/%s/%s" % (repo, revision, str(uuid.uuid4()))
```

Comment:
Reusing the job_guid will actually be okay, we might just upload the same log twice.